data/json/items/gun/45.json

```diff
@@ -84,15 +86,17 @@
     "type": "GUN",
     "name": "L39B-45",
     "description": "After the success of the 9mm model, Leadworks LLC released an L39B chambered in .45.  It soon earned a reputation as one of the most intimidating guns on the market, for firer and target alike.",
-    "ammo": "45"
+    "ammo": "45",
+    "ranged_damage": 1
   },
   {
     "id": "l_sp_45",
     "copy-from": "l_sp_9mm",
     "type": "GUN",
     "name": "L39-45",
     "description": "Capitalizing on the L39B-45's reputation as one of the most intimidating pistols on the market, Leadworks LLC manufactured a semiautomatic model as well.  Locking it semi-auto reduced cost and improved performance.",
-    "ammo": "45"
+    "ammo": "45",
+    "ranged_damage": 1
```

Comment:
Either `ranged_damage` of `1` or `0` is acceptable but as per the `description` the two items should be otherwise identical apart from their `ammo` field so as a result you should only need make changes to the `l_sp_9mm` definition.